AdminUi/src/AdminUi/ClientApp/src/app/components/quotas/identity/identity-edit/identity-edit.component.ts

```diff
@@ -198,7 +200,7 @@ export class IdentityEditComponent {
 
     isAllSelected() {
         const numSelected = this.selectionQuotas.selected.length;
-        const numRows = this.identity.quotas.length;
+        const numRows = this.identity.quotas ? this.identity.quotas.filter(i => i.source === 'Individual').length : 0;
```

Comment:
Is there ever a case where `this.identity.quotas` is null?

Reply:
yes, on the first lifecycle events of the component, which would cause an error.